models/balancer/ethereum/balancer_ethereum_schema.yml

```
@@ -167,4 +167,32 @@ models:
     columns:
       - *pool_id
       - *token_address
-      - *normalized_weight
\ No newline at end of file
+      - *normalized_weight
+
+  - name: balancer_vebal_slopes
+    meta:
+      blockchain: ethereum
+      project: balancer
+      contibutors: markusbkoch, mendesfabio, stefenon
+    config:
+      tags: ['balancer', 'ethereum', 'vebal', 'slope', 'markusbkoch', 'mendesfabio', 'stefenon']
+    description: >
+        Slope and bias of veBAL per wallet after each balance update
+        Depends on veBAL_call_create_lock, veBAL_evt_Deposit and veBAL_evt_Withdraw
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_time
+            - wallet_address
```

Also change the uniqueness test.
```suggestion
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - block_date
            - wallet_address
```